docs/essentials/collections/threadsafe/blockingCollectionOverview.md

```diff
@@ -1 +1,115 @@
+# BlockingCollection Overview
+
+[BlockingCollection&lt;T&gt;](http://dotnet.github.io/api/System.Collections.Concurrent.BlockingCollection%601.html) is a thread-safe collection class that provides the following features:
+
+*   An implementation of the Producer-Consumer pattern.
+
+*   Concurrent adding and taking of items from multiple threads.
```

Comment:
@cartermp Take is a LINQ method that returns a specified number of elements from the beginning of a sequence. In this context, we're talking about blocking a collection to prevent items from being added or removed while we're executing a Take. Looking at the feedback history on the MSDN topic I don't see anyone questioning this, so I assume it's clear to developers familiar with LINQ.